cli/package.json

```diff
@@ -21,59 +21,54 @@
         "src"
     ],
     "dependencies": {
-        "@babel/core": "^7.26.0",
-        "@babel/plugin-proposal-class-properties": "^7.8.3",
-        "@babel/plugin-proposal-nullish-coalescing-operator": "^7.8.3",
-        "@babel/plugin-proposal-optional-chaining": "^7.8.3",
+        "@babel/core": "^7.27.4",
```

Comment:
these upgrades got rid of most problems:
- a minor upgrade for babel, and adding some plugins that were "proposals" before but now they're not.
- git rid of few dependencies that were not used anywhere (and causing warnings)
- other minor upgrades that got rid of dependency warnings
- `jest/core` upgrade - this is the one upgrade that I am not sure if it has other implications (on consuming projects running tests for example - need to investigate)